helpers/helpers.v2.1.d/go

```diff
@@ -19,15 +19,17 @@
 #
 
 readonly GOENV_INSTALL_DIR="/opt/goenv"
-# goenv_ROOT is the directory of goenv, it needs to be loaded as a environment variable.
+
+# GOENV_ROOT is the directory of goenv, it needs to be loaded as a environment variable.
 export GOENV_ROOT="$GOENV_INSTALL_DIR"
+export goenv_root="$GOENV_INSTALL_DIR"
 
 _ynh_load_go_in_path_and_other_tweaks() {
 
-    # Get the absolute path of this version of go
-    go_dir="$GOENV_INSTALL_DIR/versions/$app/bin"
+    # Get the absolute path of this version of Go
+    go_dir="$GOENV_INSTALL_DIR/versions/$go_version/bin"
```

Comment:
I'm super confused because it is `$app` for the ruby helper x_X Or is the ruby helper also wrong ? https://github.com/YunoHost/yunohost/blob/dev/helpers/helpers.v2.1.d/ruby#L30

I don't understand what defines the name of these folder ... I assume they are created when calling `rbenv install` ? X_x

Reply:
Ruby helper use the alias plugins : 
https://github.com/YunoHost/yunohost/blob/f7ed895c1811d13e444e043b0c9ace6401518bbc/helpers/helpers.v2.1.d/ruby#L126
and
https://github.com/YunoHost/yunohost/blob/f7ed895c1811d13e444e043b0c9ace6401518bbc/helpers/helpers.v1.d/ruby#L184

That plugins is not available for goenv, so there is no `$app` in `$GOENV_INSTALL_DIR/versions/`